lib/WebDAV/BasicAuthBackend.php

```diff
@@ -125,9 +134,19 @@ private function login(string $username, string $password)
 
         list($user, $userPassword) = $this->loginService->login($profile);
 
+        if (PHP_SESSION_ACTIVE === session_status()) {
```

Comment:
Please add a comment here (and ideally in the `setupUserFs` function above as well) for why this check is necessary. You might also add the link to the Nextcloud issue that requires this workaround.